scripts/fetch-dashboard-assets.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
suggestion: Using build.rs in the `dashboard` crate will be more flexible. 

For example, you can implement the current bash logic into `build.rs` and use [`-F`](https://doc.rust-lang.org/cargo/commands/cargo-build.html) to control the compiling. When building the whole project,  we can decide whether to integrate dashboard assets or not:

```bash
$ cargo build -F integrate-dashboard
```

All the complexity will hide in the `dashboard` crate:

- Control the dashboard assets downloading;
- Control the dashboard upstream version;
- ...

Reply:
I still left all the dashboard assets downloading/unextract/cleaning etc in the original shell script, because I think these kinds of tasks are best suited for shell script, and people (like the frontend guys) not familiar with rust (but need to debug or change the dashboard assets related logic) can also work with it. 

That said, I've added a feature flag `dashboard` in crate "servers", and the the `fetch-dashboard-assets.sh` is now being called in `build.rs`. One can enable it by `cargo build -F servers/dashboard`, get rid of the needs to manually execute that "fetch" script.

This feature flag also resolve the comment below. If feature `dashboard` is not enabled, the endpoint `/dashboard` is not used.

Reply:
@MichaelScofield Sounds reasonable to me.